pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -84,6 +84,10 @@ var (
 		MountPath: "/builderPullSecrets",
 		ReadOnly:  true,
 	}
+	bindingsEnv = corev1.EnvVar{
+		Name:  "CNB_BINDINGS",
+		Value: "/var/bindings",
```

Comment:
It seems that [buildpacks spec](https://github.com/buildpacks/spec/blob/master/extensions/bindings.md#detect-and-build-phases) expects that this in `/platform/bindings`.

Reply:
Good catch. I was paying attention to the launch phase which is variable based.

Reply:
Moved the mount path to be under `/platform/bindings`, I need to test it to make sure the nested volume mounts don't cause issues